src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { Mode, ModeAction, ModeService } from '../../../shared/services/mode.service';
+import { ContentInfo } from '../../../shared/models/content/content-info';
+import { Observable } from 'rxjs';
+import { CurrentContentService } from '../../../shared/services/current-content.service';
+import { delay } from 'rxjs/operators';
+
+@Component({
+  selector: 'alg-content-top-bar',
+  templateUrl: './content-top-bar.component.html',
+  styleUrls: [ './content-top-bar.component.scss' ],
+})
+export class ContentTopBarComponent {
+  @Input() fullFrameContent = false;
+  @Input() currentMode?: Mode;
```

Comment:
If `modeService` is injected, I think this input should be replaced by a regular prop like `currentMode = this.modeService.mode$.asObservable()` or something like that

Plus, it would also allow to remove current mode input in `TopBarComponent` as it would not be useful anymore :grin:

Reply:
Fixed